contents/handbook/company/culture.md

```diff
@@ -6,30 +6,28 @@ showTitle: true
 
 So, what's it like working at PostHog?
 
-<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/rRwzJiljpSA" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+## We're 100% remote 
 
-## All remote
+[Our team](/people) is 100% remote, and distributed across more than 15 countries.
 
-Our [team](/people) is 100% remote, and distributed across over 10 countries.
+In addition to all the equipment you'll need, we provide [a budget to help you find coworking space](/handbook/people/spending-money#work-space), or to cover the costs of coffees for those who prefer not to work at home.
 
-As well as the equipment you'll need, we provide [a budget to help you find coworking space](/handbook/people/spending-money#work-space) or to cover the costs of coffees for those who prefer not to work at home.
-
-All remote has a bunch of advantages:
+Being all remote has a bunch of advantages:
 
 * We can hire [amazing people](/people) from a global talent pool.
-* Being remote encourages a deeper level of personal thought and writing things down.
-* It allows for uninterrupted work.
-* It makes results clearer, which helps us hold people to account for outcomes rather than hours spent in the office.
+* It encourages thoughtful, and intentional, written communication
+* It creates space for lots of uninterrupted work.
+* We judge performance based on real outcomes, not hours spend in the office.
```

Comment:
"in the office" → "in front of your computer"?

Reply:
Could change to "hours spent in an office". I think the point is begin remote eliminates presenteeism, which is specifically associated with being in an office.